repairnator/repairnator-scanner/src/main/java/fr/inria/spirals/repairnator/scanner/Launcher.java

```diff
@@ -367,6 +371,13 @@ private void printToStdout(Map<ScannedBuildStatus, List<BuildToBeInspected>> lis
                     if (this.config.getLauncherMode() == LauncherMode.REPAIR) {
                         Launcher.LOGGER.info("Incriminated project and build: " + buildToBeInspected.getBuggyBuild().getRepository().getSlug() + ":" + buildToBeInspected.getBuggyBuild().getId());
                         System.out.println(buildToBeInspected.getBuggyBuild().getId());
+
+                        // If KUBERNETES mode then build id will also be sent to ActiveMQ queue.
```

Comment:
Since you have `protected void kubernetesProcess(String slug)` method specifically for `KUBERNETES` mode, is it possible to move these code there?

Reply:
Hi, I just look at it. It would result into repeting the loops code inside printToStdout :) , so I don't think it'll improve much.

Reply:
The logic is the same, but you could make it more readable and debuggable :) `printToStdout()` should have nothing to do with `KUBERNETES` mode. Imagine later someone else will work on your code, it is easier for her to fix everything in one method :`kubernetesProcess` :)

Reply:
Just went through the original code, `printToStdout` indeed checks different builds. Then let's keep it for now, or it needs more work to polish `printToStdout` :)

Reply:
yes, sir :) .